doc/getstarted/build_and_install/docker_install_en.rst

```diff
@@ -59,73 +106,46 @@ The Jupyter Notebook is an open-source web application that allows
 you to create and share documents that contain live code, equations,
 visualizations and explanatory text in a single browser.
 
-PaddlePaddle Book is an interactive Jupyter Notebook for users and developers. 
+PaddlePaddle Book is an interactive Jupyter Notebook for users and developers.
 We already exposed port 8888 for this book. If you want to
 dig deeper into deep learning, PaddlePaddle Book definitely is your best choice.
 
-Once you are inside the container, simply issue the command:
+We provide a packaged book image, simply issue the command:
 
 .. code-block:: bash
-        
-    jupyter notebook
+
+    docker run -p 8888:8888 paddlepaddle/book:<version>
 
 Then, you would back and paste the address into the local browser:
-    
+
 .. code-block:: text
 
     http://localhost:8888/
 
 That's all. Enjoy your journey!
 
-
-Non-AVX Images
---------------
-
-Please be aware that the CPU-only and the GPU images both use the AVX
-instruction set, but old computers produced before 2008 do not support
-AVX.  The following command checks if your Linux computer supports
-AVX:
-
-.. code-block:: bash
-
-   if cat /proc/cpuinfo | grep -i avx; then echo Yes; else echo No; fi
-
-
-If it doesn't, we will need to build non-AVX images manually from
-source code:
-
-.. code-block:: bash
-
-   cd ~
-   git clone https://github.com/PaddlePaddle/Paddle.git
-   cd Paddle
-   docker build --build-arg WITH_AVX=OFF -t paddle:cpu-noavx -f paddle/scripts/docker/Dockerfile .
-   docker build --build-arg WITH_AVX=OFF -t paddle:gpu-noavx -f paddle/scripts/docker/Dockerfile.gpu .
-
-
 Development Using Docker
 ------------------------
 
 Developers can work on PaddlePaddle using Docker.  This allows
 developers to work on different platforms -- Linux, Mac OS X, and
 Windows -- in a consistent way.
 
-1. Build the Development Environment as a Docker Image
+1. Build the Development Docker Image
 
    .. code-block:: bash
 
       git clone --recursive https://github.com/PaddlePaddle/Paddle
       cd Paddle
-      docker build -t paddle:dev -f paddle/scripts/docker/Dockerfile .
-
+      docker build -t paddle:dev .
 
    Note that by default :code:`docker build` wouldn't import source
    tree into the image and build it.  If we want to do that, we need
```

Comment:
we need to set a build arg -> we need docker the development docker image and then run the following command: